extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -230,11 +260,13 @@ public void run() {
     static abstract class SimpleTrigger implements Trigger {
 
         private final String id;
+        private boolean running;
```

Comment:
We need to use a thread-safe construct here, e.g. `volatile` field + synchronized acccess.

Reply:
I set running boolean to volatile and marked getter and setter as synchronized